### PR TITLE
fix this EYE RAPING bit of code that could NEVER WORK

### PR DESCRIPTION
**Pull request by SwadicalRag**
like whoever made this was 420 high on weed

### CODE REVIEW

**Comment by notcake** [comment]
the patriarchy

**Comment by SwadicalRag** [comment]
nice

File: plugins/lua/superstring.lua [hunk 1]
```diff
@@ -74,5 +74,8 @@ local function eval_expr( expr )
 	return "{err: " .. err .. "}"
 end
 function meta:__bnot( arg )
```
Comment (zerfgog) [inline plugins/lua/superstring.lua]: :+1:

Comment (SwadicalRag) [inline plugins/lua/superstring.lua]: meta:__b1

Comment (zerfgog) [inline plugins/lua/superstring.lua]: but you should probably fix it

Comment (SwadicalRag) [inline plugins/lua/superstring.lua]: adding a space between the two functions?

**Comment by SwadicalRag** [comment]
it's about time

**Comment by mcd1992** [comment]
Use better commit messages fag

**Comment by SwadicalRag** [comment]
no

**Comment by mcd1992** [comment]
Oh, okay then.

**Pull request merged**